packages/relay/src/ServerConfigParams.ts

```diff
@@ -150,7 +152,8 @@ export const serverDefaultConfiguration: ServerConfigParams = {
   requestMinValidBlocks: 3000, // roughly 12 hours (half client's default of 6000 blocks
   runPaymasterReputations: true,
   coldRestartLogsFromBlock: 1,
-  pastEventsQueryMaxPageSize: Number.MAX_SAFE_INTEGER
+  pastEventsQueryMaxPageSize: Number.MAX_SAFE_INTEGER,
+  recentActionAvoidRepeatDistanceBlocks: 10
```

Comment:
As I suggested over the phone, can we use `confirmationsNeeded`? They seem to match, instead of having another config param for it.

Reply:
seems to be ok (I'm all for less config param), except that the REASON for them is different: confirmations is used to protect against uncles/forks, and this param against network instabilities (which in turn, and the source for forks, so it might be ok..)

Reply:
The reason for them is quite similar actually, finality issues when getting an answer from the node. You separate them because you know that there is a difference on the rpc node for the reason of the inconsistent answer, but from the relayer PoV, it waits this amount of blocks as confirmations to further the chance of finality. It doesn't matter whether it's because of a node out of sync with other nodes on the network, or a uncles/forks. The reason for the server is the same.